containers/elabctl/elabctl.sh

```
@@ -0,0 +1,687 @@
+    if [ -e $CONF_FILE ]; then
+        echo 55 | dialog --backtitle "$backtitle" --title "$title" --gauge "Making a copy of the existing configuration file." 20 80
+        \cp $CONF_FILE ${CONF_FILE}.old
+    fi
+
+    # get a config file already filled with random passwords/keys
+    curl --silent "https://get.elabftw.net/?config" -o "$TMP_CONF_FILE"
+    sleep 1
+
+    # elab config
+    echo 50 | dialog --backtitle "$backtitle" --title "$title" --gauge "Adjusting configuration" 20 80
+    sed -i -e "s/SERVER_NAME=localhost/SERVER_NAME=$servername/" $TMP_CONF_FILE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)